src/system/clients/client/profile_change_client/profile_change_client.rs

```diff
@@ -0,0 +1,18 @@
+use crate::prelude::*;
+
+#[derive(Debug)]
+pub struct ProfileChangeClient {
```

Comment:
This is the v1 client that notifies the hosts that profile change, this will change as we get to the final version of SargonOS, where we would expose granular updates. But for the sake of easier migration - this is the first/intermediary step.